tests/scripts/task_sphinx_precheck.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Precheck if sphinx docs build can fail.
+set -e
+set -u
+set -o pipefail
+
+cleanup()
+{
+  rm -rf /tmp/$$.*
+}
+trap cleanup 0
+
+# cleanup cache
+rm -rf docs/tutorials
+rm -rf docs/vta/tutorials
+find . -type f -path "*.pyc" | xargs rm -f
+
+echo "PreCheck sphinx doc generation WARNINGS.."
+cd docs
+TVM_TUTORIAL_EXEC_PATTERN=none make html 2>/tmp/$$.log.txt
```

Comment:
Should we call `make clean` here directly so that we can remove line 31-32?

Reply:
```make clean``` by sphinx does not remove the generated gallery folders

Reply:
I see. Never mind.